Amazon.IonObjectMapper/Factory/IIonSerializerFactory.cs

```diff
@@ -16,26 +16,21 @@ namespace Amazon.IonObjectMapper
     using System.Collections.Generic;
 
     /// <summary>
-    /// This interface is to create a custom IonSerializer to be used to serialize
-    /// and deserialize instances of the class annotated with Factory IonSerializerAttribute.
+    /// Interface for creating a custom IonSerializer to be used to serialize/deserialize
+    /// instances of classes annotated with IonSerializerAttribute's Factory.
     /// </summary>
     public interface IIonSerializerFactory
     {
         /// <summary>
         /// Create custom IonSerializer with customContext option.
         /// </summary>
-        /// <param name="options">
-        /// The IonSerializationOptions is an object that can be passed to the IonSerializer object
-        /// and determined the way to customize the IonSerializer.
-        /// </param>
+        ///
+        /// <param name="options">Serialization options for customizing serializer behavior.</param>
         /// <param name="customContext">
-        /// The customContext is one option to create IonSerializer with custom arbitrary data.
-        /// A Dictionary of Key Type string is to map to any customized objects
```

Comment:
I think the old docs are more clear for customContext

Reply:
I've expanded my explanation a bit.